sumo/wordnet

```diff
@@ -0,0 +1 @@
+wn
```

Comment:
I think this is a symlink?  (github makes it hard to tell)  This would be the first and only symlink in use in this project.  (Despite the cargo cult `Options` used everywhere)  I think it would work, but maybe just using `.htaccess` rules is easier to understand than using file system features.  Since this is setting up new redirects, and the readme even says to use `wn`, does this name and/or symlink style need to be used?

Reply:
Hi @davidlehn  thanks for the comment.  The symlink is not actually needed since this is new W3ID. And I'd prefer to follow existing practices.  I have updated commit to remove the unnecesary symlink. thanks